packages/infra/medplum.test.config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "apiPort": 8103,
+  "name": "test",
+  "region": "us-west-2",
+  "accountNumber": "819438846128",
+  "stackName": "MedplumTest",
+  "domainName": "medplum-test.letsdevelo.com",
+  "apiDomainName": "api.medplum-test.letsdevelo.com",
+  "appDomainName": "app.medplum-test.letsdevelo.com",
+  "storageDomainName": "storage.medplum-test.letsdevelo.com",
+  "storageBucketName": "medplum-test-storage",
+  "maxAzs": 2,
+  "rdsInstances": 1,
+  "desiredServerCount": 1,
+  "serverMemory": 512,
+  "serverCpu": 256,
+  "serverImage": "medplum/medplum-server:2.0.1",
```

Comment:
Pinning this to v2.0.1 avoids the bug in medplum server code that causes the R4 endpoint to fail.

The `:latest` tag is updated extremely frequently. And when @hankedori stopped and restarted the fargate task over the weekend, that caused fargate to pull the newest medplum-server image (which has a bug). This is why things broke. Moving back to an older version of medplum-server restores functionality